# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375249168 -7200
#      Wed Jul 31 07:39:28 2013 +0200
# Node ID 110842a29d1b837cc019ce5e93c9e8f96b43a869
# Parent  6c56ff28be2850152d879e0156f64771ba3858f2
-Fix (r20670): Right padding of the object class string should be negative.

diff --git a/src/object_gui.cpp b/src/object_gui.cpp
--- a/src/object_gui.cpp
+++ b/src/object_gui.cpp
@@ -210,7 +210,7 @@ public:
 					if (objclass->GetUISpecCount() == 0) continue;
 					if (!this->vscroll->IsVisible(pos++)) continue;
 					SetDParam(0, objclass->name);
-					DrawString(r.left + WD_MATRIX_LEFT, r.right + WD_MATRIX_RIGHT, y + WD_MATRIX_TOP, STR_JUST_STRING,
+					DrawString(r.left + WD_MATRIX_LEFT, r.right - WD_MATRIX_RIGHT, y + WD_MATRIX_TOP, STR_JUST_STRING,
 							((int)i == _selected_object_class) ? TC_WHITE : TC_BLACK);
 					y += this->line_height;
 				}
